src/components/bottom-bar/bettom-bar-utils.js

```diff
@@ -0,0 +1,39 @@
+//@flow
+
+import {PlaybackControls, PLAYER_BREAK_POINTS} from 'components';
+
+const SPATIAL_CONTROLS = {TimeDisplayPlaybackContainer: 107};
+
+// eslint-disable-next-line require-jsdoc
+export function calculateControlsSize(controls: string[], currentControlWidth: number, playerWidth: number, isPlaylist: boolean): number {
+  let totalWidth = 0;
+  let controlWidth = 0;
+  for (let control of controls) {
+    if (control in SPATIAL_CONTROLS) {
+      controlWidth = SPATIAL_CONTROLS[control];
+    } else if (control === PlaybackControls.displayName) {
+      if (playerWidth > PLAYER_BREAK_POINTS.SMALL) {
+        controlWidth = isPlaylist ? currentControlWidth * 3 : currentControlWidth;
```

Comment:
playlist is the only case in which we need this ?

Reply:
Right , only playlist has 2 additional controls (next & prev)